### ISSUE TITLE
jigsaw - improve report files

### ISSUE THREAD

**Issue by ascbot** [DOI-USGS/ISIS3#2047]
---

Author Name: **Lynn Weller** (Lynn Weller)

---

Jigsaw creates a number of text files with a wealth of information in a variety of formats.  Some simple improvements to these reports would greatly benefit the user.

*bundleout.txt:*
- Add input image list, maybe right after "Network Filename"

- Add the "FILE_PREFIX" value, maybe near input network and image list filenames?

- Add titles (w/ units where applicable?) to "IMAGE MEASURES SUMMARY"

- Add units to values under "IMAGE EXTERIOR ORIENTATION ***J2000***"
   Ken and I talked about this some...probably best to add units next to Image Parameter name as opposed to "Value" and "Correction" columns since units will
   be different between camera and spacecraft values and will vary for position, velocity and acceleration output.

*bundleout_images.csv*
- Add units for applicable columns

*bundleout_points.csv*
- Add columns for corrections in decimal degrees (reported in bundleout.txt, but not here); this should not replace the Corrections in meters

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Lynn Weller** (Lynn Weller)
Original Date: 2015-08-14T19:09:02Z

---

I have related this post to #1005 which points out that when the parameter overexisting (as well as overhermite) is used, the "Initial Value" is set to 0.0 as opposed to something meaningful.  This apparently is not a trivial fix and is therefore being kept as a separate post, but I include here since it pertains to the bundleout text files.

**Comment by jessemapel** [comment]
@lwellerastro Are these recommendations covered by #3227

**Comment by lwellerastro** [comment]
@jessemapel, in reference to #3227 specifically, 

bundleout.txt:

Add input image list, maybe right after "Network Filename"
**Yes**

Add the "FILE_PREFIX" value, maybe near input network and image list filenames?
**Yes**

Add titles (w/ units where applicable?) to "IMAGE MEASURES SUMMARY"
**Yes**

Add units to values under "IMAGE EXTERIOR ORIENTATION J2000"
Ken and I talked about this some...probably best to add units next to Image Parameter name as opposed to "Value" and "Correction" columns since units will
be different between camera and spacecraft values and will vary for position, velocity and acceleration output.
**Yes!**  
Actually, this last one had been added some time ago, it was the units for accuracy that was recently updated.

I took a look at bundleout_points.csv and it contains units as requested above (likely added some time ago).  bundleout_images does not have units, but I think that will be painful since there are so many options depending on what was solved for.  We've lived without them for this long and that info is now in the bundleout.txt, so let's not worry about that one. 

This post can be closed if that's what you're after.
